### DISCUSSION TITLE
Cannot successfully connect to ikev2 vpn server on Windows

### DISCUSSION THREAD

**Discussion by pencilnav** [strongswan/strongswan#895] [Q&A]
The vpn server i am connecting to has both a Server IP and Server ID, i've sucessfully connected to it via IKEv2 EAP UID+PWD on android, linux console, macos and ios devices (android is with the strongswan app, ios is with the system's vpn setup). However when i try setting up on windows devices it's either with IP only and receive ike credentials incorrect (i am sure the certificate and UID+PWD is correct) or with Server ID and no response.
i've checked with the vpn provider and they cannot figure out what's the issue. Also i cant really find that FAQ has the answer.

**Comment by tobiasbrunner** [comment]
The IP/hostname you configure in the GUI has to be confirmed by the certificate, so either add the IP address as subjectAltName extension to the server certificate or make the server ID a FQDN that resolves to the proper IP so you can configure that.

**Reply by tobiasbrunner** [reply]
So you are not actually using strongSwan but connect to a commercial VPN service? Then please ask them for help.
